app/HutchManager/Controllers/AgentsController.cs

```diff
@@ -7,15 +7,19 @@ namespace HutchManager.Controllers;
 
 [ApiController]
 [Route("api/[controller]")]
-[AllowAnonymous]
+[Authorize]
```

Comment:
Shouldn't need this explicitly; all controllers in the codebase default to requiring the default authorization policy
```suggestion
```

Reply:
I was able to query without authenticating, thus included. I will remove it as suggested.